app/models/allocation.rb

```diff
@@ -3,6 +3,8 @@ class Allocation < ApplicationRecord
   belongs_to :accredited_body, class_name: "Provider"
   belongs_to :recruitment_cycle
 
+  audited
```

Comment:
Shouldn't we associate this with the provider?

Reply:
```suggestion
  audited associated_with: :provider
```

Reply:
I checked earlier
```
=> #<Audited::Audit:0x00007fea288ac6d0
 id: 259515,
 auditable_id: 436,
 auditable_type: "Allocation",
 associated_id: nil,
 associated_type: nil,
 user_id: 8916,
 user_type: "User",
 username: nil,
 action: "update",
 audited_changes: {"request_type"=>[1, 2], "number_of_places"=>[28, 0]},
 version: 2,
 comment: nil,
 remote_address: "::1",
 request_uuid: "3c498ddc-5048-4214-a819-28cebe7370fd",
 created_at: Thu, 04 Jun 2020 10:07:41 UTC +00:00>
```
The user and allocation is sufficient as that associates with everything that we need